core/cli/src/informant.rs

```diff
@@ -82,9 +83,13 @@ pub fn start<C>(service: &Service<C>, exit: ::exit_future::Exit, handle: TaskExe
 				(proc.cpu_usage(), proc.memory())
 			} else { (0.0, 0) };
 
+			let network_state = serde_json::to_string(&network.network_state())
+				.expect("NetworkState implements Serialize, JSON serialization should work; qed");
```

Comment:
That's not a very good explanation, but I don't know what to write here. The error exists only because serde is too generic.

Reply:
I'd typically write `"Serializing to String never errors; qed"`

Reply:
I stumbled across this some time ago, I had a hashmap that was not using a String as key. Compiling worked, but executing returned an error :O

Reply:
So maybe return the error here?

Reply:
How about using `unwrap_or_default`, the default being an empty string?